.circleci/config.yml

```diff
@@ -48,15 +48,53 @@ jobs:
           command: npm run test:integration
       - store_artifacts:
           path: ./screenshots
-  site-deploy:
+
+  # This pushes our deployable assets to a branch, for SRE to use in deploying to stage and prod.
+  deploy-branch-update:
     docker:
       - image: cimg/node:15.1
     steps:
       - attach_workspace:
           at: .
+      - run:
+          name: Set up ssh known_hosts # https://circleci.com/docs/2.0/gh-bb-integration/
+          command: |
+            mkdir -p ~/.ssh
+            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
```

Comment:
Would it make sense to set this as an enviroment variable?

Reply:
I did it this way because it's exactly what circle's doc suggests, I figured that'd be clearest for anyone stumbling across it :)